SPECS/grub2/grub2.spec

```diff
@@ -326,7 +340,19 @@ cp $GRUB_PXE_MODULE_SOURCE $EFI_BOOT_DIR/$GRUB_PXE_MODULE_NAME
 %{_libdir}/grub/*
 %endif
 
+%files rpm-macros
+%dir %{_sysconfdir}/default/grub.d
```

Comment:
the grub.d dir should belong to the normal grub rpm, not the rpm-macros rpm

Reply:
I think we want to put this directory in rpm-macros because every package which wants to maintain configurations in there needs to install a package containing /etc/default/grub.d.

If we install that file via grub2, systems that don't need all of its bloat will end up taking it anyways.